src/pages/Setup/Stepper.tsx

```diff
@@ -0,0 +1,34 @@
+import { useTranslation } from "react-i18next";
+
+const NUMBER_OF_STEPS = 4;
+
+type Props = {
+  currentStep: number;
+};
+
+export default function Stepper({ currentStep }: Props) {
+  const { t } = useTranslation();
+
+  return (
+    <div className="w-full">
+      <ul className="lg:flex hidden justify-center items-center gap-4">
+        {Array.from({ length: NUMBER_OF_STEPS }).map((_, index: number) => (
+          <>
+            <li
+              className={`list-none text-center flex flex-col ${index < currentStep ? "opacity-100" : "opacity-30"}`}
+              key={index}
+            >
+              <span className="font-bold">Step {index + 1}</span>
+              <span className="text-sm">
+                {t(`setup.step.step_${index + 1}_description`)}
+              </span>
+            </li>
+            {index < NUMBER_OF_STEPS - 1 && (
```

Comment:
Not sure about this. Looks not semantical.  Should be to only be `li`s within `ul` I believe.
Could the line be achieved with i.e. [`:after`](https://stackoverflow.com/a/75512805/1667461) on the `ul`?
Like make it a full with block element with a border or background, position it absolute maybe and cover it with the li's?
Do you know what I mean?

Reply:
Tried to do it with `after` and stuff like `:not(:last-child)` to not target the last `li`.
Unfortunately didn't work out as expected, so I left it at that.

Reply:
Styling the `ul` seemed like a good idea at first, but then you would need to style all the `li` and the line would be too close to the elements itself.

Reply:
Could give it a try as well. There must be nicer way :)

Reply:
Should we merge this one then? In the end, it's looking good & the 3 extra elements do not put a burden on performance or anything.